javascripts/discourse/components/modal/insert-video.js

```diff
@@ -2,10 +2,13 @@ import Component from "@glimmer/component";
 import { tracked } from "@glimmer/tracking";
 import { action } from "@ember/object";
 import { empty } from "@ember/object/computed";
+import { inject as service } from "@ember/service";
```

Comment:
```suggestion
import { service } from "@ember/service";
```
This is the modern version of the import 😎